build/cdc/dev/Dockerfile.nl-python.cdc.dev

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Development Dockerfile for Custom Data Commons NL server
+
+# Use official Python runtime as a parent image
+FROM python:3.12-slim
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the NL requirements files to the container
+COPY server/requirements.txt /app/server_requirements.txt
+COPY nl_server/requirements.txt /app/nl_server_requirements.txt
+COPY nl_requirements.txt /app/nl_requirements.txt
+
+# Create the python virtual environment and install dependencies
+RUN python3 -m venv /app/.env
+RUN /app/.env/bin/pip install --upgrade pip setuptools wheel
+RUN /app/.env/bin/pip install --only-binary :all: pandas==2.1.1
+RUN /app/.env/bin/pip install --only-binary :all: scikit-learn==1.3.1
```

Comment:
If these come packaged with python, do they need to be installed?

Reply:
Good call-removed